src/components/ConferencePanel/index.js

```diff
@@ -0,0 +1,173 @@
+import React, { PropTypes, Component } from 'react';
+import formatNumber from 'ringcentral-integration/lib/formatNumber';
+import IconField from '../IconField';
+import Switch from '../Switch';
+import i18n from './i18n';
+import styles from './styles.scss';
+import RcFont from '../../assets/RcFont/RcFont.scss';
+
+class ConferencePanel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showInternational: false,
+      searchInternationals: this.props.conferenceNumbers.phoneNumbers,
+      selectInternationals: [],
+    };
+    this.formatPin = (number) => {
+      if (!number) {
+        return '';
+      }
+      return number.replace(/(\d{3})/g, '$1-').replace(/-$/, '');
+    };
+    this.formatNumbers = {
+      dialInNumber: formatNumber({
+        phoneNumber: this.props.conferenceNumbers.phoneNumber,
+        countryCode: this.props.countryCode,
+        areaCode: this.props.areaCode || '',
+      }),
+      hostCode: this.formatPin(this.props.conferenceNumbers.hostCode),
+      participantCode: this.formatPin(this.props.conferenceNumbers.participantCode)
+    };
+    this.onInternationalSwitch = (checked) => {
+      this.setState({
+        showInternational: checked,
+      });
+    };
+    this.onSearchKeyUp = (e) => {
+      const searchKey = e.currentTarget.value;
+      this.setState({
+        searchInternationals: this.getMatchList(searchKey)
+      });
+    };
+    this.getMatchList = (searchKey) => {
+      const key = searchKey.toLowerCase().trim();
+      return this.props.conferenceNumbers.phoneNumbers.filter(value =>
+        value.phoneNumber.indexOf(key) >= 0 ||
+        value.country.name.toLowerCase().trim().indexOf(key) >= 0);
+    };
+    this.inviteWithText = () => {
+      let inviteText = 'Please join the RingCentral conference.\n\n';
```

Comment:
The text should be specified in i18n. Can reference code [here](https://github.com/ringcentral/ringcentral-js-widget/blob/master/src/components/CallingSettingsAlert/index.js#L17).